agent/config/config.go

```diff
@@ -533,6 +533,7 @@ func environmentConfig() (Config, error) {
 		TaskCPUMemLimit:                     parseBooleanDefaultTrueConfig("ECS_ENABLE_TASK_CPU_MEM_LIMIT"),
 		DockerStopTimeout:                   parseDockerStopTimeout(),
 		ContainerStartTimeout:               parseContainerStartTimeout(),
+		ContainersPullUpfront:               parseBooleanDefaultTrueConfig("ECS_PULL_DEPENDENT_CONTAINERS_UPFRONT"),
```

Comment:
lets call this `DepContainersPullUpfront` or `DependentContainersPullUpfront` to convey the full meaning

Reply:
Renamed it to `DependentContainersPullUpfront`. Thanks!